src/MahApps.Metro/Controls/Dialogs/InputDialog.cs

```diff
@@ -278,21 +282,18 @@ protected override void OnLoaded()
             this.Icon = this.DialogSettings.Icon;
```

Comment:
Is it not better to move this to the base class? Because it is used in each derived class. So a user can use this in a custom dialog too. WDYT?